Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry.fedoraproject.org/fedora:29
+
+# Pass the release version from to build this docker image.
+ARG relver
+
+LABEL maintainer="Factory 2 Team" \
+      description="A microservice triggered by specific message to tag a build."
+
+RUN dnf update -y && \
+    dnf install -y rpm-devel python3-devel krb5-devel gcc redhat-rpm-config && \
+    dnf clean all
+
+RUN python3 -m venv prodenv && \
+    prodenv/bin/pip install message-tagging-service==${relver}
```

Comment:
`--no-deps` ?

Reply:
This virtualenv is created without `--system-site-packages`. The deps are still required to be installed.

Reply:
Sure, but if you miss to install a dependency before calling this command, it's going to fetch it from pypi which we'd like to avoid. The no deps parameter ensures that doesn't happen.

Reply:
This is fixed.